packages/react-scripts/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": "^10.12.0 || >=12.0.0"
+    "node": ">=14"
```

Comment:
Is it necessary for us to declare this package as being only node 14 and above?
I understand we are not using ensure-node-env so this will only produce a warning when being installed.
My understanding though is since this is being released as 9.6.0 - we expect users of node 12 and future adopters of node 16 to be using this.

Since node-sass 6.0.1 can be used on node 12, is it sufficient to leave the node engine as `^10.12.0 || >=12.0.0`?

Reply:
> Is it necessary for us to declare this package as being only node 14 and above? I understand we are not using ensure-node-env so this will only produce a warning when being installed. My understanding though is since this is being released as 9.6.0 - we expect users of node 12 and future adopters of node 16 to be using this.
> 
> Since node-sass 6.0.1 can be used on node 12, is it sufficient to leave the node engine as `^10.12.0 || >=12.0.0`?

Thanks @michael-booth for your suggestion, I think we should remove '^10.12.0' and set as '>=12.0.0', because node-sass 6.0.1 can't be used on node 10, how do you think of this?

<img width="970" alt="image" src="https://user-images.githubusercontent.com/52264893/154401030-a6362ff7-33ee-40d0-bcd6-8de7da7db046.png">

Reply:
Yes agreed @SophiaChen2021